test/cctest/test_node_postmortem_metadata.cc

```diff
@@ -53,7 +53,7 @@ TEST_F(DebugSymbolsTest, BaseObjectPersistentHandle) {
 
   auto expected = reinterpret_cast<uintptr_t>(&obj.persistent());
   auto calculated = reinterpret_cast<uintptr_t>(&obj) +
-      nodedbg_offset_BaseObject__persistent_handle_;
+      NODEDBG_OFFSET(BaseObject, persistent_handle_, v8_Persistent_v8_Object);
```

Comment:
I think it will help legibility when the variable name is written out rather than going through a macro.  It's not wrong as such, just harder to parse for humans.